rusk-wallet/src/bin/main.rs

```diff
@@ -36,6 +38,7 @@ use io::{prompt, status, WalletArgs};
 pub(crate) struct WalletFile {
     path: WalletPath,
     pwd: Vec<u8>,
```

Comment:
```suggestion
    aes_key: Vec<u8>,
```

Since this is no longer the password, I would name it key or aes_key. I would also add the IV as a field here, given that the salt is now a field as well.

This is also more in line with how the web wallet stores these values.
For reference:
```ts
type WalletEncryptInfo = {
  data: Uint8Array;
  iv: Uint8Array;
  salt: Uint8Array;
};
```